instrumentation-appender-api/src/main/java/io/opentelemetry/instrumentation/appender/api/DelegatingLogEmitterProvider.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.appender.api;
```

Comment:
WDYT about using `...instrumentation.api.appender`? Or even `...instrumentation.api.logs.appender`; the appender api would under the same parent package as `LoggingContextConstants` then

Reply:
I flipped it, but didn't add the `logs` level, will add this to list of things to revisit.